let a = 1 in raise(RuntimeException)